pkg/providers/common/provider.go

```diff
@@ -32,6 +32,9 @@ func (p *Provider) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(
 		&p.BuildVersion, "build-version", "", "Kubernetes Build Version",
 	)
+	flags.StringVar(
+		&p.Runtime, "runtime", "", "Runtime used while installing k8s cluster",
```

Comment:
Let's set the `docker` as a default value as of now, will change to containerd or something in future..

Reply:
The variable in k8s-ansible has the default value as docker.
As we dont have default values here for variables s3_server, bucket, directory, I thought we needn't have for runtime as well.

Reply:
@mkumatag ^^

Reply:
make sense